src/components/search-result-list/search-result.list.tsx

```diff
@@ -12,7 +13,10 @@ const SearchResultList: React.FC<SearchResultListProps> = ({ resultItems }) => {
     <ul className="search-result-page__list my-32">
       {resultItems.map((item, i) => (
         <li key={item.workId}>
-          <SearchResultListItem item={item} coverTint={getCoverTint(i)} />
+          <SearchResultListItem
+            item={item as Work}
```

Comment:
Could `resultItems` also be instances of `Work` instead of `WorkSmallFragment` so we avoid the `as`?

Reply:
Yes skipped it initially because I ended up in a rabbit hole. But now I found my way out